source/lib-intro.tex

```diff
@@ -3039,12 +3039,18 @@
 #define MB_CUR_MAX @\seebelow@
 
 namespace std {
+  // Exposition-only function type aliases
+  extern "C" using c_exit_t = void();                                // \expos
```

Comment:
[objects.within.classes]/2 only defines what the "// exposition only" comment means for private members of classes. This isn't the only place where we would be using it for something else, but it might be a good idea to move the definition of "exposition only" to somewhere more general, and just refer to it from [objects.within.classes]/2.